packages/backend/src/workers/provider/LlamaCppPython.ts

```diff
@@ -77,13 +78,15 @@ export class LlamaCppPython extends InferenceProvider {
       throw new Error('The model info file provided is undefined');
     }
 
+    console.log('path:', modelInfo.file.path, ' file:', modelInfo.file.file);
+
     const labels: Record<string, string> = {
       ...config.labels,
       [LABEL_INFERENCE_SERVER]: JSON.stringify(config.modelsInfo.map(model => model.id)),
     };
 
     // get model mount settings
-    const filename = getLocalModelFile(modelInfo);
+    const filename = modelInfo.file.path + path.sep + modelInfo.file.file;
```

Comment:
question: should getLocalModelFile be updated to return the right value using path.sep ?

Reply:
https://github.com/containers/podman-desktop-extension-ai-lab/blob/e2d5a2e6b5839337575fbd622ef271ca1d46be1b/packages/backend/src/utils/modelsUtils.ts#L29-L32

I don't see the difference between using `path.sep` and `path.join` 🤔  we should not have differences ??

Reply:
Explanation here https://github.com/containers/podman-desktop-extension-ai-lab/issues/2366#issuecomment-2592119657

Reply:
Fixing the model mounting on platform that upload it https://github.com/containers/podman-desktop-extension-ai-lab/pull/2385